### DISCUSSION TITLE
INITIAL PROPOSAL: Implement replicated security onto the Cronos PoS chain and have the Cronos EVM chain be validated by the PoS validators

### DISCUSSION THREAD

**Discussion by Terra854** [crypto-org-chain/chain-main#1021] [Ideas]
Hello everyone,

I would like to suggest that the Cronos team implement Replicated Security for Cronos PoS chain. We can take a page out of ATOM's replicated security and have the Cronos PoS validators in the active set validate the Cronos EVM chain at the same time instead of having it being limited to invite-only

More info on Cosmos Replicated Security:
https://www.reddit.com/r/cosmosnetwork/comments/117yebf/what_is_replicated_security/
https://forum.cosmos.network/t/preparing-for-replicated-security/8925
https://forum.cosmos.network/t/proposal-187-accepted-v9-lambda-upgrade-with-replicated-security/8766

**Comment by raulillo82** [comment]
Could we get any feedback from development team?

**Reply by yihuang** [reply]
we are trying to use interchain-security to connect Cronos PoS chain and Cronos.

**Reply by Terra854** [reply]
Sounds good @yihuang . If possible, please try to give regular updates on the progress on that

**Reply by Terra854** [reply]
Hi there @yihuang , checking in to see if progress is made on this

**Reply by Terra854** [reply]
Hi there @yihuang any updates on this? It has been a long while

**Reply by Terra854** [reply]
Hi there @yihuang , is this still under consideration? Cause validators will need a revenue source once the CRO supply reaches 100B
